curations/gem/rubygems/-/github-auth.yaml

```diff
@@ -6,3 +6,9 @@ revisions:
   3.0.0:
     licensed:
       declared: MIT
+    files:
+      - attributions:
+           - 'Copyright (c) 2013 Chris Hunt'
+        license:  MIT
+        path: https://github.com/chrishunt/github-auth/blob/master/LICENSE.txt
```

Comment:
The `path` value should be the path to the file being curated. For this package, the license file `path` would be `LICENSE.txt` since it is in the root of the package.

It should look something like this:
```
    files:
      - attributions:
           - Copyright (c) 2013 Chris Hunt
        license: MIT
        path: LICENSE.txt
```

Here is a another gem package with file level curations for reference: https://github.com/clearlydefined/curated-data/blob/master/curations/gem/rubygems/-/calabash-android.yaml